scripts/md2rst.py

```diff
@@ -27,7 +27,10 @@ def download_pandoc():
 
 
 def md2rst(text):
+    input_format = os.environ.get('FORMAT', 'markdown_github')
```

Comment:
FORMAT seems like a very generic name. We should use such a name that conveys it's primary usage just from it's name. Please use a more specific name like MARKDOWN_FLAVOUR or something like that.